src/main/java/build/buf/protovalidate/internal/constraints/DescriptorMappings.java

```diff
@@ -215,8 +215,21 @@ static Type getCELType(FieldDescriptor fieldDescriptor, boolean forItems) {
           return Decls.newWellKnownType(Type.WellKnownType.DURATION);
         case "google.protobuf.Timestamp":
           return Decls.newWellKnownType(Type.WellKnownType.TIMESTAMP);
+        case "google.protobuf.BytesValue":
+          return Decls.newWrapperType(Decls.Bytes);
+        case "google.protobuf.DoubleValue":
+        case "google.protobuf.FloatValue":
+          return Decls.newWrapperType(Decls.Double);
+        case "google.protobuf.Int32Value":
+        case "google.protobuf.Int64Value":
+          return Decls.newWrapperType(Decls.Int);
+        case "google.protobuf.StringValue":
+          return Decls.newWrapperType(Decls.String);
+        case "google.protobuf.UInt32Value":
+        case "google.protobuf.UInt64Value":
+          return Decls.newWrapperType(Decls.Uint);
```

Comment:
Explanation: In most cases, the wrapper type is "unwrapped" by having its constraints processed in `processWrapperConstraints`; in other cases (e.g. nested in a message), it is handled by cel-java. However, in the case of the `rule`, we might be dealing with a wrapper or repeated wrapper directly. Therefore,`getCELType` needs to be aware of wrappers.